src/universal/modules/userDashboard/containers/UserActionList/UserActionListContainer.js

```diff
@@ -43,35 +68,70 @@ const mapStateToProps = (state) => {
     filter: {
       actions: actionFilterFn,
       teams: teamsFilterFn
-    }
+    },
+    mutationHandlers
   }).data;
   return {
     actions,
     teams,
     userId: state.auth.obj.sub,
-    selectingNewActionTeam: state.userDashboard.selectingNewActionTeam
+    selectingNewActionTeam: state.userDashboard.selectingNewActionTeam,
+    queryKey
   };
 };
 
+// function getNewIndex(actions, sourceSortOrder, targetSortOrder) {
```

Comment:
Remove this commented section before merge? Or, add a comment as to what its future might be?